CRM/MembershipExtras/Service/PaymentMethod.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+use Civi\Api4\EntityFinancialAccount;
+
+class CRM_MembershipExtras_Service_PaymentMethod {
```

Comment:
This is not a service, it just a class that creates a payment method, you should instead move it to this folder:
https://github.com/compucorp/uk.co.compucorp.membershipextras/tree/a09d5ea648c9c7e1bec98f7910343dc4dc541a13/CRM/MembershipExtras/Setup/Manage

also probably you should rename it to:

`CRM_MembershipExtras_Setup_Manage_OfflineRecurringPaymenMethod` instead of `CRM_MembershipExtras_Setup_Manage_PaymenMethod`
so it is more clear what this calss does, once you do that you can actually reduce this constant name from:
MANUAL_PAYMENT_PROCESSOR_PAYMENT_METHOD_NAME

to just

NAME